src/crd/policies/cluster_admission_policy.rs

```diff
@@ -131,6 +131,11 @@ pub struct ClusterAdmissionPolicySpec {
     /// the policy is assigned to.
     #[serde(default)]
     pub context_aware_resources: Vec<ContextAwareResource>,
+
+    /// Message is  used to specify the message that will be returned when
+    /// the policy is rejected. The specific policy results will be
+    /// returned in the warning field of the response.
```

Comment:
This has to be updated to reflect what I've written on [this PR](https://github.com/kubewarden/kubewarden-controller/pull/1140#discussion_r2142607502)